src/richie/apps/search/views.py

```diff
@@ -24,3 +26,19 @@ def bootstrap_elasticsearch(request, version):
 
     messages.info(request, _("The search index was successfully bootstrapped"))
     return Response({})
+
+
+@api_view(["GET"])
+# pylint: disable=unused-argument
+def filter_definitions(request, version):
```

Comment:
Since this only changes when we change settings, it could be calculated only once at startup instead of calling it each time the view is called.